pwnlib/heap/glmalloc/utils.py

```diff
@@ -0,0 +1,33 @@
+import re
+from pwnlib.util import packing
+
+
+def u32(bytes_):
+    return packing.u32(bytes_)
+
+
+def u64(bytes_):
+    return packing.u64(bytes_)
+
+
+def p32(number):
+    return packing.p32(number)
+
+
+def p64(number):
+    return packing.p64(number)
```

Comment:
Just get rid of all this and use `from pwnlib.util.packing import *`

Reply:
Done in f965dc6